wallet/test/Test/Pos/Wallet/Web/Tracking/SyncSpec.hs

```diff
@@ -42,7 +42,7 @@ spec :: Spec
 spec = withCompileInfo def $ withDefConfigurations $ \_ -> do
     describe "Pos.Wallet.Web.Tracking.BListener" $ modifyMaxSuccess (const 10) $ do
         describe "Two applications and rollbacks" twoApplyTwoRollbacksSpec
-    describe "Pos.Wallet.Web.Tracking.evalChange" $ do
+    xdescribe "Pos.Wallet.Web.Tracking.evalChange (pending, CSL-2473)" $ do
```

Comment:
@parsonsmatt Weird, I remember a fix on this test from @avieth a while ago. Am I getting confused?

Reply:
Nope, it's just *still* broken. I've commented another seed in the relevant YouTrack issue.